### DISCUSSION TITLE
When using prisma to process a mysql database, @DateTime@default (now()) generates a date 8 hours later than the current date

### DISCUSSION THREAD

**Discussion by StrongerLph** [prisma/prisma#24284] [Q&A]
### Bug description

When using prisma to process a mysql database, @DateTime@default (now()) generates a date 8 hours later than the current date.
![image](https://github.com/prisma/prisma/assets/47054165/e3b8ab96-0a9f-4ec6-8c39-4846d92f4e37)
![image](https://github.com/prisma/prisma/assets/47054165/9e921749-88f3-487b-bc0d-64cc552f7e79)
The expected result is : 2024-05-23 17:37:30, But what we actually get is this 2024-05-23 09:37:30

This is my database time configuration:
![image](https://github.com/prisma/prisma/assets/47054165/777fdf64-1334-484b-8963-65f68887c9f2)

```sql
SELECT NOW()    -- get 2024-05-23 17:37:30
```

Is this a problem with the framework itself?
How should I handle this? If there is a problem with the frame itself, when can it be fixed?

### How to reproduce

```prisma
model EmailLog {
  id               BigInt   @id @default(autoincrement())
  messageId        String   @unique(map: "message_id_index") @map("message_id") @db.VarChar(255)
  sender           String   @db.VarChar(255)
  recipient        String   @db.VarChar(1000)
  subject          String   @db.VarChar(255)
  content          String   @db.Text
  attachments      String   @db.VarChar(255)
  hitConfiguration String   @map("hit_configuration") @db.VarChar(255)
  createBy         String?  @default("") @map("create_by") @db.VarChar(64)
  createTime       DateTime @default(now()) @map("create_time") @db.DateTime(0)
  updateBy         String?  @default("") @map("update_by") @db.VarChar(64)
  updateTime       DateTime @default(now()) @map("update_time") @db.DateTime(0)

  @@map("rpa_email_auto_log")
}
```

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by ludralph** [comment]
Hi @StrongerLph 👋 

Thank you for raising this question.

Prisma currently saves `DateTime` in UTC, which might not align with your local timezone settings. This can cause issues when your MySQL database is configured with a non-UTC timezone. See this [issue](https://github.com/prisma/prisma/issues/5051). You can also see how a user  worked around this issue [here](https://github.com/prisma/prisma/discussions/23070#discussioncomment-8444680).

**Reply by StrongerLph** [reply]
Ok, thanks,I've looked at similar issues, but is prisma able to provide more fine-grained configuration of DateTime in the future, such as time zone configuration, as this can cause similar problems when connecting to some older databases, and it would be more flexible to specify a time zone. My current approach to the auto-generated DateTime is to remove it from the schema so that the assignment of this field is completely handled by mysql.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
